### DISCUSSION TITLE
Where are shape locations defined?

### DISCUSSION THREAD

**Discussion by duxonline** [OrchardCMS/OrchardCore.Commerce#297] [General] [answered]
I'm trying to understand how OrderPartDisplayDriver works. Take the following code as an example, could anyone tell me where the locations like "Content" and "Meta" defined? I assumed they were defined in Layout.cshtml, but I could not find them in any theme layout files.  Thank you for you input.

```   
    public override IDisplayResult Display(OrderPart part, BuildPartDisplayContext context) =>
        Initialize<OrderPartViewModel>(GetDisplayShapeType(context), viewModel => PopulateViewModelAsync(viewModel, part))
            .Location("Detail", "Content:25")
            .Location("Summary", "Meta:10");
```

**Comment by sarahelsaig** [comment, marked as answer]
Content and Meta are local zones in the content item, they are in the Content template file [here](https://github.com/OrchardCMS/OrchardCore/blob/main/src/OrchardCore.Modules/OrchardCore.Contents/Views/Content.cshtml#L16). If you aren't familiar with Orchard Core's display management, you can read more [in the documentation](https://docs.orchardcore.net/en/latest/docs/reference/core/Placement/) or watch [the related video tutorial](https://www.youtube.com/watch?v=YR8QzyAEgo4) (this is part of a comprehensive OC course [here](https://www.youtube.com/watch?v=aSNRH-BDAdY&list=PLuskKJW0FhJcSX7j0Bd-1X5hq3dgCtYwO)).

If you have other questions about Orchard Core, the best place to ask is the repository's Gitter room [here](https://gitter.im/OrchardCMS/OrchardCore?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).

**Comment by duxonline** [comment]
Thank you @DAud-IcI. I had actually read the doc and watched the video you mentioned. It's local zone part that confused me.
